docs/OCP_CI_Tutorials/Step_Registry/Step_Registry_Guide.md

```diff
@@ -0,0 +1,53 @@
+# OpenShift CI Step Registry Guide<!-- omit from toc -->
+
+## Table of Contents<!-- omit from toc -->
+- [Introduction](#introduction)
+- [Step Types](#step-types)
+  - [Refs](#refs)
+  - [Chains](#chains)
+  - [Workflows](#workflows)
+- [The interop Folder](#the-interop-folder)
+  - [Tooling](#tooling)
+  - [Scenarios](#scenarios)
+
+## Introduction
+This document is meant to serve as a general guide to the step registry in OpenShift CI. It will cover:
+- The difference between the types of steps
+- The `interop` folder in the step registry
+
+> **NOTE:**
+> 
+> This is guide is not meant to replace the official OpenShift CI documentation, but rather act as supplemental documentation for how the CSPI team plans to use these steps. Please see the [official OpenShift CI documentation](https://docs.ci.openshift.org/docs/architecture/ci-operator-internals/steps/) for additional information.
+
+## Step Types
+The step registry is a folder in the [openshift/release](https://github.com/openshift/release) repository that stores steps that can be used during test execution. For the purposes of this document, we will only cover refs, chains, and workflows as those are the steps primarily used by the CSPI team at the time of writing this document.
+
+### Refs
+A ref is the most basic step in the step registry, it can be thought of as the building-blocks of other steps. A ref is essentially a BASH script that be used on it's own or as part of a chain or workflow. 
```

Comment:
```suggestion
A ref is the most basic step in the step registry, it can be thought of as the building-blocks of other steps. A ref is essentially a BASH script that can be used on it's own or as part of a chain or workflow. 
```